pkg/utils/pod.go

```diff
@@ -196,3 +196,23 @@ func GetContainerIdFromPod(pod *v1.Pod, containerName string) string {
 	}
 	return ""
 }
+
+// Whether pod uses ext resource ext-resource.node.gocrane.io, and value it uses
+func ExtResourceDetect(pod *v1.Pod, resName v1.ResourceName) (hasExtResource bool, extCpuLimit, extCpuRequest int64) {
+	for i := range pod.Spec.Containers {
+		container := pod.Spec.Containers[i]
+		for res, val := range container.Resources.Limits {
+			if strings.HasPrefix(res.String(), fmt.Sprintf(ExtResourcePrefixFormat, resName)) {
+				hasExtResource = true
+				extCpuLimit += val.MilliValue()
```

Comment:
How about if we set the ext resource to 0 ?

Reply:
if strings.HasPrefix(res.String(), fmt.Sprintf(ExtResourcePrefixFormat, v1.ResourceCPU)) && val.Value() != 0